components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/resources/userstore.yaml

```diff
@@ -593,6 +593,9 @@ components:
         description:
           type: string
           example: Some description of the user store
+        isLocalUserStore:
```

Comment:
shall we add a description to this property?

Reply:
fixed